test/cmd/basicresources.sh

```diff
@@ -299,9 +300,10 @@ os::test::junit::declare_suite_start "cmd/basicresources/projectadmin"
 temp_config="$(mktemp -d)/tempconfig"
 os::cmd::expect_success "oc config view --raw > '${temp_config}'"
 export KUBECONFIG="${temp_config}"
-os::cmd::expect_success 'oc policy add-role-to-user admin test-user'
-os::cmd::expect_success 'oc login -u test-user -p anything'
-os::cmd::try_until_success "oc project '$(oc project -q)'"
+os::cmd::expect_success 'oc policy add-role-to-user admin project-admin'
+os::cmd::expect_success 'oc login -u project-admin -p anything'
+os::cmd::expect_success 'oc new-project test-project-admin'
+os::cmd::try_until_success "oc project test-project-admin"
```

Comment:
Also what does the `KUBECONFIG` changes and login done here change for the tests below this in `basicresources`? Why should they inherit these changes?